packages/fxa-auth-server/lib/routes/utils/signup.js

```diff
@@ -64,6 +65,8 @@ module.exports = (log, db, mailer, push, verificationReminders) => {
       // Our post-verification email is very specific to sync,
       // so only send it if we're sure this is for sync.
       if (service === 'sync') {
+        const onMobileDevice =
+          userAgent(request.headers['user-agent']).deviceType === 'mobile';
         const mailOptions = {
```

Comment:
I would pass the `onMobileDevice` value into the `mailerOptions` and pass that into `sendPostVerifyEmail`.